graphkb/genes.py

```diff
@@ -444,6 +444,10 @@ def get_gene_information(
     gene_flags["oncogene"] = convert_to_rid_set(get_oncokb_oncogenes(graphkb_conn))
     logger.info("fetching tumour supressors list")
     gene_flags["tumourSuppressor"] = convert_to_rid_set(get_oncokb_tumour_supressors(graphkb_conn))
+    logger.info("fetching cancer_gene list")
+    gene_flags["cancer_gene"] = convert_to_rid_set(get_cancer_genes(graphkb_conn))
```

Comment:
Hi @mathieulemieux  - did you  check that IPR is already able to accept 'cancer_gene'?

Hi @Nithriel, @mathieulemieux 
Is there a reason why 'cancer_gene' is not camel case like all the other gene flags?  eg. (cancerRelated, tumourSuppressor)

Reply:
>did you check that IPR is already able to accept 'cancer_gene'?
Work needed to be done on both side, gkb and ipr, and I don't thing it matters which one first, as long as we make sure it's done on both sides before release. @Nithriel should tell us when the work on her side is done.

>Is there a reason why 'cancer_gene' is not camel case like all the other gene flags?
I thought we agreed on that before. I can change it to camel case if not too late from the ipr side. @Nithriel , what is the exact flag you're using on your side?

Reply:
cancer_gene changed for concerGene.